packages/apidom-ls/src/services/validation/linter-functions.ts

```diff
@@ -993,4 +995,23 @@ export const standardLinterfunctions: FunctionItem[] = [
       return true;
     },
   },
+  {
+    functionName: 'apilintOpenAPIPathTemplateWellFormed',
+    function: (element: Element, strict = false) => {
+      if (isStringElement(element)) {
+        const pathTemplate = toValue(element);
+        return test(pathTemplate, { strict });
+      }
+      return true;
+    },
+  },
+  {
+    functionName: 'apilintOpenAPIPathTemplateValid',
+    function: (element: Element) => {
```

Comment:
This is where you need to implement the logic of matching template expressions against the Parameter Objects.